303-quarkus-vertx-sql/src/main/java/io/quarkus/qe/vertx/sql/services/FlightSearchService.java

```diff
@@ -37,7 +39,7 @@ void onStart(@Observes StartupEvent ev) {
         return Flight.findByOriginDestination(connection, query.from, query.to)
                 .onItem()
                 .transform(flight -> calculatePrice(query, flight))
-                .collectItems().in(ArrayList::new, List::add);
+                .collect().in(ArrayList::new, List::add);
```

Comment:
why this you change this method?

Reply:
`.collectItems` is deprecated by `.collect`.